kubernetes_state/check.py

```diff
@@ -217,17 +223,37 @@ def kube_job_complete(self, message, **kwargs):
         for metric in message.metric:
             tags = []
             for label in metric.label:
-                tags.append(self._format_tag(label.name, label.value))
+                trimmed_job = self._trim_job_tag(label.value)
+                tags.append(self._format_tag(label.name, trimmed_job))
```

Comment:
Apologies. I misunderstood your original comment, but as I read it again, I see what you're talking about. 

Thanks for your patience